moveit_core/planning_scene/CMakeLists.txt

```diff
@@ -47,21 +42,14 @@ if(BUILD_TESTING)
 
   ament_add_gtest(test_planning_scene test/test_planning_scene.cpp
     APPEND_LIBRARY_DIRS "${append_library_dirs}")
-  target_link_libraries(test_planning_scene
-    ${MOVEIT_LIB_NAME}
-    ${srdfdom_LIBRARIES}
-    ${urdf_LIBRARIES}
-    ${geometric_shapes_LIBRARIES}
-    ${OCTOMAP_LIBRARIES}
-    ${ament_index_cpp_LIBRARIES}
+  ament_target_dependencies(test_planning_scene
+    ament_index_cpp
+    geometric_shapes
+    srdfdom
   )
+  target_link_libraries(test_planning_scene ${MOVEIT_LIB_NAME})
 
   ament_add_gtest(test_multi_threaded test/test_multi_threaded.cpp
     APPEND_LIBRARY_DIRS "${append_library_dirs}")
-  target_link_libraries(test_multi_threaded
-    moveit_planning_scene
-    moveit_test_utils
-    ${MOVEIT_LIB_NAME}
-    ${urdfdom_LIBRARIES}
-    ${urdfdom_headers_LIBRARIES})
+  target_link_libraries(test_multi_threaded moveit_test_utils ${MOVEIT_LIB_NAME})
```

Comment:
`robot_state`, `robot_model` and `collision_detection` are included, but not listed here?

Reply:
same